spearhead/functions.php

```diff
@@ -245,3 +245,61 @@ function spearhead_the_excerpt( $excerpt ) {
 
 // Filter the excerpt
 add_filter( 'get_the_excerpt', 'spearhead_the_excerpt' );
+
+/*
+ * Post footer meta
+ */
+
+if ( ! function_exists( 'seedlet_entry_meta_footer' ) ) :
+	/**
+	 * Prints HTML with meta information for the categories, tags and comments.
+	 */
+	function seedlet_entry_meta_footer() {
+
+		seedlet_posted_on();
+
+		// Edit post link.
+		edit_post_link(
+			sprintf(
+				wp_kses(
+					/* translators: %s: Name of current post. Only visible to screen readers. */
+					__( 'Edit <span class="screen-reader-text">%s</span>', 'spearhead' ),
+					array(
+						'span' => array(
+							'class' => array(),
+						),
+					)
+				),
+				get_the_title()
+			),
+			'<span class="edit-link">',
+			'</span>'
+		);
+	}
+endif;
+
+if ( ! function_exists( 'seedlet_posted_on' ) ) :
+	/**
+	 * Prints HTML with meta information for the current post-date/time.
+	 */
+	function seedlet_posted_on() {
```

Comment:
Should this be called `spearhead_posted_on`?

Reply:
I thought so initially but if we rename it, we need to call again the new functions on their respective templates. Unless there's a better way of doing that that I'm not aware of.